src/settings-ui/Settings.UI/SettingsXAML/Views/GeneralPage.xaml

```diff
@@ -362,6 +362,26 @@
                         IsTabStop="{x:Bind ViewModel.IsExperimentationGpoDisallowed, Mode=OneWay}"
                         Severity="Informational" />
                 </controls:SettingsGroup>
+                <controls:SettingsGroup x:Uid="General_DiagnosticsAndFeedback" Visibility="Visible">
+                    <StackPanel>
+                        <HyperlinkButton
+                            x:Uid="GeneralPage_DiagnosticsAndFeedback_Link"
+                            Margin="-8,0,0,0"
+                            NavigateUri="https://aka.ms/powertoys-data-and-privacy-documentation" />
+                        <InfoBar
+                            x:Uid="DataDiagnostics_SettingIsGPOManaged_InfoBar"
```

Comment:
Please use the existing Uid for single gpo settings that we can reuse the existing resw string.